lib/src/main/java/uk/gov/di/ipv/core/library/service/CiStorageService.java

```diff
@@ -100,25 +132,186 @@ public void submitMitigatingVcList(
     public List<ContraIndicatorItem> getCIs(
             String userId, String govukSigninJourneyId, String ipAddress)
             throws CiRetrievalException {
+        InvokeResult result =
+                invokeClientToGetCIResult(
+                        CI_STORAGE_GET_LAMBDA_ARN,
+                        govukSigninJourneyId,
+                        ipAddress,
+                        userId,
+                        "Retrieving CI's from CI storage system.");
+        String jsonResponse = new String(result.getPayload().array(), StandardCharsets.UTF_8);
+        GetCiResponse response = gson.fromJson(jsonResponse, GetCiResponse.class);
+        return response.getContraIndicators();
+    }
+
+    public ContraIndications getContraIndicatorsVC(
```

Comment:
I think, can method name getContraIndicatorsVCFromCimit()

Reply:
this return VC with contraIndicators